conanfile.py

```diff
@@ -20,11 +20,10 @@ class TaraxaConan(ConanFile):
 
     def _add_clang_utils_on_darwin(self):
         current_path = os.getcwd()
-        source_path = os.path.dirname(current_path)
         clang_format = "clang-format"
         clang_tidy = "clang-tidy"
-        path_to_format = source_path + "/" + clang_format
-        path_to_tidy = source_path + "/" + clang_tidy
+        path_to_format = current_path + "/" + clang_format
+        path_to_tidy = current_path + "/" + clang_tidy
```

Comment:
I changed this to source directory instead of build directory. Because sometimes need to clean up the build directory for re-building. In that case, don't need to download clang-format and clang-tidy again

Reply:
But that path is to conanfile, not to build dir...
Okay, I will double check that

Reply:
I think now it fixed=)